neard/src/cli.rs

```diff
@@ -465,6 +465,7 @@ impl RunCmd {
                 debug!(target: "neard", "{} server stopped", name);
             }))
             .await;
+            opentelemetry::global::shutdown_tracer_provider(); // Finish sending spans.
```

Comment:
This is perhaps too early point to shutdown opentelemetry -- we are yet to `Drop` a bunch of stuff, I'd imagine, and we can conceivably do some o11y in drops. Perhaps a better place would be a Drop of an object returned here:

https://github.com/near/nearcore/blob/c6204b1d8f6067b07b6632c200bac5c1354bff88/neard/src/cli.rs#L36-L40

?

Reply:
You have a good point. Moved the shutdown to be the last operation in the `block_on()` call.

Batch span processor requires tokio runtime to be active.
If I move the shutdown out of the `block_on()` function call, then I get 1 or multiple errors:
```
OpenTelemetry trace error occurred. cannot send span to the batch span processor because the channel is closed
```

Reply:
Another option would be to do another `runtime.block_on()` call at the end of `parse_and_run()`, but that is also not possible, because `runtime` was already consumed by `actix::System::with_tokio_rt()`.

Reply:
Ah, indeed!

I don't think that moving to the last op necessary helps: the drops I think happen in `System.run()` after that. But I don't see a good solution here.